app/models/user.rb

```diff
@@ -58,6 +58,14 @@ def self.from_oauth_hash(auth_hash)
     user
   end
 
+  def client_model
+    Proposal.client_model_for(self)
+  end
+
+  def client_model_slug
+    client_model.to_s.underscore.tr("/", "_")
```

Comment:
how is this different from the `client_slug` method already available on a `user` ?